prow/plugins/lgtm/lgtm.go

```diff
@@ -306,7 +306,7 @@ func handle(wantLGTM bool, config *plugins.Configuration, ownersClient repoowner
 				log.WithError(err).Errorf("Failed to get pull request.")
 			}
 			if pr.MergeSHA != nil {
```

Comment:
`MergeSHA` is still used in a few places in this file. It should be replaced everywhere with the `Head.SHA` and the nil checks can be removed.